guides/plugins/plugins/storefront/override-existing-javascript.md

```diff
@@ -101,6 +101,17 @@ const PluginManager = window.PluginManager;
 PluginManager.override('CookiePermission', MyCookiePermission, '[data-cookie-permission]');
 ```
 
+::: info
+If the plugin you want to override is an async plugin, the import of your override plugin has to be a async as well. See also [Registering an async plugin](./add-custom-javascript.md#registering-an-async-plugin)
```

Comment:
```suggestion
If the plugin you want to override is an async plugin, then the import of your override plugin has to be an async as well. See also [Registering an async plugin](./add-custom-javascript.md#registering-an-async-plugin)
```

Reply:
Whoops, the "a" was not intended. It was supposed to be:

```
... plugin has to be async as well.
```

Reply:
I changed it. ✔️